redash/handlers/users.py

```diff
@@ -37,10 +37,17 @@
 )
 
 
-def invite_user(org, inviter, user):
+def invite_user(org, inviter, user, send_email=True):
+    email_configured = settings.MAIL_DEFAULT_SENDER is not None
+    d = user.to_dict()
+
     invite_url = invite_link_for_user(user)
-    send_invite_email(inviter, user, invite_url, org)
-    return invite_url
+    if email_configured and send_email:
```

Comment:
We should return `invite_link` _only_ if `email_configured=False`. In this case, if I pass `send_email=False`, I can get an invite link even if email server is configured.

This means that in a secure environment like the hosted service, I could create a user with any email address I want and get an invite link to it without receiving the actual email address.

Reply:
Good catch. For some reason I thought that the no-invite flag returns the invite link, but it actually doesn't.